rust-toolchain.toml

```diff
@@ -1,2 +1,2 @@
 [toolchain]
-channel = "1.72.1"
+channel = "1.75"
```

Comment:
A small comment on this: the version of the Rust toolchain is linked to the version of Zenoh. The reason being that Zenoh-Flow can be compiled to a Zenoh plugin and, in order for Zenoh to load the plugin, it needs to have been compiled with the same version of Rust.

Hence, if you want to tackle this part, I'd suggest to make a separate PR that does both: bump Zenoh to version 1.0.x and update the toolchain to 1.75 😉

(I did not look closely at the rest of your PR as you marked it as a draft)

Reply:
Oh thx! Yeah bumping Zenoh was also in my mind, I should definitely separate it in two PR.